rpcs3/Emu/RSX/VK/vkutils/device.cpp

```diff
@@ -586,9 +584,15 @@ namespace vk
 		// enabled_features.shaderCullDistance = VK_TRUE;  // Alt notation of clip distance
 
 		enabled_features.samplerAnisotropy = VK_TRUE;
-		enabled_features.textureCompressionBC = pgpu->optional_features_support.texture_compression_bc;
+		enabled_features.textureCompressionBC = pgpu->features.textureCompressionBC;
 		enabled_features.shaderStorageBufferArrayDynamicIndexing = VK_TRUE;
 
+		if (!pgpu->features.textureCompressionBC && (get_driver_vendor() == driver_vendor::V3DV || get_driver_vendor() == driver_vendor::PANVK))
```

Comment:
This will crash the driver again. You want to add the check for our variables, not the driver's.
Add back the "or" on line 135 and it should work.
We use those optional_features flags to decide what to emulate vs use hardware.